packages/sdk/src/actions/getSuggestedFees.ts

```diff
@@ -150,10 +150,7 @@ export async function getSuggestedFees({
 }: GetSuggestedFeesParams) {
   const data = await fetchAcrossApi<SuggestedFeesResponse>(
     `${apiUrl}/suggested-fees`,
-    {
-      depositMethod: "depositExclusive",
```

Comment:
Hmm, that is a tricky one 🤔 The API is not updated yet because it depends on upgraded SpokePools... We should probably keep this `depositMethod` param for now. If the API is upgraded, it will ignore this param anyways.

Reply:
The thing is right now, we need to explicitly specify `depositMethod: "depositExclusive",` so that it behaves the same as before. After the API change gets merged (which needs to happen after the SpokePool upgrade), there is no need for this param anymore. So in order to keep the current behavior, we need still need that param

Reply:
Okay so we will need to API updates to go in before we can release this then, because CI will fail using current prod API. because we are getting exclusivity parameters that will cause reverts in the new upgraded contract.